htmresearch/algorithms/apical_tiebreak_bayesian_temporal_memory.py

```diff
@@ -133,19 +133,19 @@ def __init__(self,
     self.basalBias = np.zeros((self.numBasalSegments, self.columnCount*self.cellsPerColumn))
     self.apicalBias = np.zeros((self.numApicalSegments, self.columnCount*self.cellsPerColumn))
 
-    # Initialize moving averages with 1/(M_i*M_j), 1/(M_i) or 1/(M_j)
+    # Initialize moving averages uniformly with 1/(M_i*M_j), 1/(M_i) or 1/(M_j)
     self.basalMovingAverages = np.full(
-      (self.numBasalSegments, self.columnCount*self.cellsPerColumn, self.basalInputSize),
```

Comment:
Let's use `np.zeros` instead. Or we use the previous initialisation if it hasn't caused issues. We can justify this with Sandberg's thesis whereas this one is more a mere guess? 
This might be also a potential candidate for testing. So if we use `np.zeros` here we can implement in a next feature a flag to use different initialisations

Reply:
Although, haven't you written in the description of the PR that the weights are explicitly set to 1?

Reply:
They are explicitly set to 1 if they go above that (exploding weights/bias).
Initialization is currently again set to 0 because that lead to a better testing, but can be tested around with. Thats why its kept to np.full to be easily changable. We can adapt the code also to flag for that later if we want.